src/grocy.rs

```diff
@@ -319,3 +322,36 @@ fn prompt_location(grocy_state: &GrocyState, default_location: u32) -> Result<Lo
 
     Ok(location)
 }
+
+fn prompt_barcode_userfields(grocy_api: &GrocyApi) -> Result<HashMap<String, String>> {
+    let userfields = grocy_api.get_barcode_userfields()?;
+    let mut values = HashMap::new();
+
+    for field in userfields {
+        let msg = format!("{} for this barcode:", field.caption);
+        let options = field
+            .config
+            .as_deref()
+            .map(|config| config.lines().map(|l| l.to_owned()).collect());
+
+        let value: Option<String> = match field.r#type.as_str() {
+            "checkbox" => dynprompt::prompt(Confirm::new(&msg), field.input_required)?,
```

Comment:
![image](https://github.com/diogotcorreia/lidl-to-grocy/assets/7467891/ec320751-ca07-4fab-8489-4efa5f3d5aaf)
The prompt doesn't say anything about it being `yes` or `no`, only after submitting an invalid answer does it tell you that.
Perhaps add a help message/placeholder saying it's a yes/no field.